docs/products/postgresql/howto/pgaudit-logging.rst

```diff
@@ -0,0 +1,114 @@
+Use PGAudit for logging on Aiven for PostgreSQL®
+================================================
+
+Discover the PostgreSQL® audit logging extension and its capabilities. Learn how to enable PGAudit on your Aiven for PostgreSQL service and how to access and visualize your logs.
+
+About PGAudit
+-------------
+
+`PGAudit <https://www.pgaudit.org/#>`_ is a PostgreSQL extension that allows comprehensive session and object audit logging. PGAudit is particularly useful when you need detailed logs for state, financial, or ISO certification audits, where *audits* are official inspections conducted by independent institutions authorized to examine specific subjects.
+
+What distinguishes PGAudit from other logging facilities is the level of detail it supports for logs. With a regular logging tool, you can get a list of all the operations performed against the database, which is good enough for monitoring purposes. However, audits often require getting to specific statements as requested by auditors, which is what PGAudit helps you achieve.
+
+Set up PGAudit
+--------------
+
+Prerequisites
+'''''''''''''
+
+* Aiven for PostgreSQL version 11 or higher
+* ``avnadmin`` role for enabling and configuring PGAudit
+* :doc:`Aiven CLI </docs/tools/cli>` / ``psql``
+
+Enable audit logs
+'''''''''''''''''
+
+.. note::
+
+    Configuration changes you are about to make take effect only on new connections.
+
+To configure PGAudit, use the ``aiven-extras`` extension and its ``set_pgaudit_parameter()`` function on a per-database level.
+
+1. Use :doc:`Aiven CLI </docs/tools/cli>` (or :doc:`psql </docs/products/postgresql/howto/connect-psql>`) to connect to your instance.
+
+   .. code-block:: bash
+
+      avn service cli --project $PG_PROJECT $PG_SERVICE_NAME
+
+2. Enable ``pgaudit`` and ``aiven-extras`` extensions.
+
+   .. code-block:: bash
+
+      CREATE EXTENSION pgaudit;
```

Comment:
Just for consistency we could write
`CREATE EXTENSION pgaudit CASCADE` here
In reality I don't think there is any dependency which needs to be installed for this extension and aiven_extras at least for now.